spec/models/rent_spec.rb

```diff
@@ -177,107 +164,85 @@
         end
       end
       context :when_first_council_second_not do
-         it 'invalid when d_til is within good rent' do
-           overlap = FactoryGirl.create(:good_rent,:with_council)
-           rent = FactoryGirl.build(:good_rent)
-           rent.d_from = overlap.d_from - 5.hours
-           rent.d_til = overlap.d_til - 5.hours
-           expect(rent).to_not be_valid
-         end
-         it 'add error if d_til within saved' do
-           overlap = FactoryGirl.create(:good_rent,:with_council)
-           rent = FactoryGirl.build(:good_rent)
-           rent.d_from = overlap.d_from - 5.hours
-           rent.d_til = overlap.d_til - 5.hours
-           rent.valid?
-           expect(rent.errors.get(:d_from)).to include("överlappar med annan bokning")
-         end
-         it 'invalid when d_from and d_til is within saved' do
-           overlap = FactoryGirl.create(:good_rent,:with_council)
-           rent = FactoryGirl.build(:good_rent)
-           rent.d_from = overlap.d_from + 5.hours
-           rent.d_til = overlap.d_til - 5.hours
-           expect(rent).to_not be_valid
-         end
-         it 'invalid when d_from is within saved rent' do
-           overlap = FactoryGirl.create(:good_rent,:with_council)
-           rent = FactoryGirl.build(:good_rent)
-           rent.d_from = overlap.d_til - 5.hours
-           rent.d_til = overlap.d_til + 5.hours
-           expect(rent).to_not be_valid
-         end
-         it 'valid when d_from and d_til is outside saved rent' do
-           overlap = FactoryGirl.create(:good_rent,:with_council)
-           rent = FactoryGirl.build(:good_rent)
-           rent.d_from = overlap.d_til + 5.hours
-           rent.d_til = rent.d_from + 5.hours
-           expect(rent).to be_valid
-         end
+        let(:overlap) { create(:rent, :good, :with_council) }
+        let(:rent) { build(:rent, :good) }
+        it 'invalid when d_til is within good rent' do
+          rent.d_from = overlap.d_from - 5.hours
+          rent.d_til = overlap.d_til - 5.hours
+          expect(rent).to_not be_valid
+        end
+        it 'add error if d_til within saved' do
+          rent.d_from = overlap.d_from - 5.hours
+          rent.d_til = overlap.d_til - 5.hours
+          rent.valid?
+          expect(rent.errors.get(:d_from)).to include("överlappar med annan bokning")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.